README_Tiflow.md

```diff
@@ -4,18 +4,14 @@
 
 This repo implements a new prototype of distributed task scheduler.
 
-### Master
+### ServerMaster
```

Comment:
Why do you change `Master` to `ServerMaster`?

Reply:
Fix it.

Reply:
Have we made the decision of replacing `servermaster` with `master`?

Reply:
The `servermaster` is an internal module name and the `master` is a role name. In my opinion, there is no conflict between them.